conference/react-js/src/reducers/forgon-password-error-message.js

```diff
@@ -0,0 +1,17 @@
+import actions from '../constants/actions-types';
+
+const { EMAIL_IS_EMPTY, EMAIL_NOT_FOUND, HIDE_EMAIL_ERROR } = actions;
+
+function forgotPasswordErrorMessage(initialstate, action) {
+  const { type } = action;
```

Comment:
Here the same situation like in ```forgot-password-error-text.jsx```.
You can return value directly from "if" statement.